ns-app/scratch/webrtc_test/trace_player.cc

```diff
@@ -41,16 +40,24 @@ void TracePlayer::LoadTrace() {
     auto uplink_traces = j["uplink"]["trace_pattern"];
     std::vector<TraceItem> traces;
     traces.reserve(uplink_traces.size());
+
+    TraceItem last_available_value;
```

Comment:
Does the variable of "last_available_value" is duplicate?
If one TraceItem don't have key loss, then ti.loss_rate_ == null. 
And in the line of device setting, the loss of device will keep with the value which affect last.
The same issue to rtt.

Reply:
it's useful.
Because NS3 don't support to set bandwidth to be zero, I have to change the the loss rate to be 100% to simulate that scenario. So how can I recover the previous loss rate if the loss rate isn't provided in some trace item? I need to guarantee the loss rate field always to be set, so this variable is for this reason.